doc/tutorial/plugins/libdnf5-plugins.rst

```diff
@@ -0,0 +1,128 @@
+.. _libdnf5 plugins tutorial:
+
+LIBDNF5 Plugins
+===============
+
+These plugins enable changes to the existing dnf5 workflow.
+Modify the library's behavior at defined points, implement new logic,
+or trigger the loading of additional plugins by implementing prepared
+callbacks.
+
+Writing the Plugin
+------------------
+
+Similar to the :ref:`dnf5 plugins tutorial`, we have to implement
+the ``libdnf5::plugin::IPlugin`` interface and override the hooks
+where additional logic is inserted within the dnf5 application lifecycle.
```

Comment:
It is a bit convoluted in the end. I would like to simplify it like:

"Similarly to the plugins tutorial, we have to implement the `libdnf5::plugin::IPlugin` interface and override the hooks to alter DNF5's logic."